### DISCUSSION TITLE
Cannot login into github

### DISCUSSION THREAD

**Discussion by GithubIsBroken** [community/community#34535] [Other Feature Feedback, Questions, & Ideas]
When i try to log in with my normal details it says my account doesn't exist, when i try the forgotten password it says the email is not registered.

The last thing i had done on the account was delete a repository, now the account is gone.

I thought i was a spanner, so tried reregistering the email again, this it says the account and email do exist. 

Ive had to create the account in order post anything. as I've had no contact or reply when i use the contact us page.

**Comment by jfullstackdev** [comment]
maybe this but there is no assurance

https://support.github.com/contact/account

**Reply by GithubIsBroken** [reply]
Didnt work, they never even responded.

**Reply by queenofcorgis** [reply]
@GithubIsBroken - please sit tight and be patient. Our Support team is working through tickets in the order your request was received.
